Sources/TensorFlow/Operators/Comparison.swift

```diff
@@ -20,80 +20,80 @@ infix operator .==: ComparisonPrecedence
 infix operator .!=: ComparisonPrecedence
 
 public extension Tensor where Scalar: Numeric & Comparable {
-    /// Computes `lhs < rhs` element-wise and returns a `Tensor` of Boolean /// scalars.
+    /// Returns a `Tensor` of Boolean scalars by computing `lhs < rhs` element-wise.
```

Comment:
Other comments mention "tensor" instead of backquoted `Tensor`. How about making it consistent here?
```suggestion
    /// Returns a tensor of Boolean scalars by computing `lhs < rhs` element-wise.
```

Same for other changes in this file.

Reply:
@rxwei I thought the same, but the commenting style is extremely uneven, some tensors have `Tensor` and some don't.